src/lib/components/BodyRenderer/Blocks/testimonial.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+  import type { TestimonialElement } from '$components/BodyParser/blocks';
+  import { getContext } from 'svelte';
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+
+  export let item: TestimonialElement;
+
+  $: themeContext = getContext('theme');
```

Comment:
```suggestion
  let themeContext = getContext('theme');
```

the context is already reactive